libraries/AsyncUDP/src/AsyncUDP.cpp

```diff
@@ -682,8 +682,8 @@ void AsyncUDP::_recv(udp_pcb *upcb, pbuf *pb, const ip_addr_t *addr, uint16_t po
         if(_handler) {
             AsyncUDPPacket packet(this, this_pb, addr, port, netif);
             _handler(packet);
-        }
-        pbuf_free(this_pb);
+        } else {
+            pbuf_free(this_pb);
```

Comment:
missed closing brace?
you in fact reverting the previous change. there was an issue with this though...